src/daemon/daemon.cpp

```diff
@@ -2527,6 +2554,12 @@ try
 
     const auto& instance_targets = instance_selection.operative_selection;
     status = cmd_vms(instance_targets, [this](auto& vm) {
+        if (vm.current_state() == VirtualMachine::State::unavailable)
```

Comment:
Hmm, restart used to fail when the instance wasn't running. Why would we introduce this new case?

States are already handled in `reboot_vm` (though the error is now wrong https://github.com/canonical/multipass/issues/4314). We could argue that we should still restart other instances (do the state filtering first), but that would be a separate matter.